### ISSUE TITLE
Create a standard documentation source that explains how to handle the label element accessibility issues on widgets and wrappers with hidden input

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#2397]
Create a standard documentation source that explains how to handle the <label> accessibility issues on widgets and MVC wrappers with hidden input (ComboBox, DropDownList, MultiSelect, NumericTextBox)

**Comment by ggkrustev** [comment]
I would like to share some details about a research that we conducted. The goal was to improve DropDownList accessibility, namely to announce selected value on focus without opening the popup. The research was inspired by TicketID: 1071774.

The DropDownList widget does not use `role="combobox"`, because this role requires an internal element with "role=textbox" (the element should be **focusable**). This cannot be done in our case due to the specifics of the DropDownList rendering. That is why the widget received the "role=listbox" attribute.

We took the time to investigate the suggested change, usage of role="combobox", in the WAI-ARIA attributes, but what we find was not so promising:

- Widget with role="combobox" attribute does not announce the selected value [NVDA output](http://screencast.com/t/cBimKzyOp).
- Widget with role="combobox" and internal <span role="textbox" aria-autocomplete="listbox"> behaves like in the previous setup [NVDA result](http://screencast.com/t/1kG2dFdhC6).

In both cases, the reader speaks out only the type of the component. The main reason is that the "textbox" control is not focusable. Unfortunately, we cannot make the internal <span> element that holds the selected text focusable.

If we go crazy and try to make the element that holds the selected text a `role="option"` control, then the reader will read the text. The unacceptable side effect is that the reader will notify the user that length of the list is "1 of 1" [NVDA output](http://screencast.com/t/eA5p6Ypt).

Based on the conducted tests, I still believe that "role=listbox" usage is correct in the context of the DropDownList widget. What we could do in order to improve our behavior and allow the reader to read out the selected value is to:
- add aria-label attribute to the DropDownList widget - http://dojo.telerik.com/OzOFI
- open the popup on focus - http://dojo.telerik.com/IZEpi

**Comment by ggkrustev** [comment]
A feasible workaround for the DropDownList was suggested in the Ticket: 1071774:
- `title` + `aria-labelledby` - http://dojo.telerik.com/ePuso

**Comment by Dimitar-Goshev** [comment]
There is such a resource in [Section 508](https://docs.telerik.com/kendo-ui/accessibility/section-508#label-element-support) article. Probably this is not an intuitive place to search for this feature...
